pkg/v1/cli/command/core/plugin_manager.go

```diff
@@ -366,7 +366,11 @@ var upgradePluginCmd = &cobra.Command{
 
 		versionSelector := repo.VersionSelector()
 		err = cli.UpgradePlugin(pluginName, plugin.FindVersion(versionSelector), repo)
-		return
+		if err != nil {
+			return err
+		}
+		log.Successf("successfully upgraded plugin %s", pluginName)
```

Comment:
Nit: to be consistent with the other messages, "successfully" -> "Successfully" (with a capital letter)

Oh but now I see that the existing message in this function does not do that...
@chandrareddyp opened #1945 about such inconsistencies .

Reply:
I think the guideline is - "every line should start with a small letter only", so we are good here.

Reply:
@marckhouzam I remember that while logging errors, we want to start the sentence with a small letter. I'll edit all success messages in this PR to start with a small letter